packages/expo-media-library/android/src/main/java/expo/modules/medialibrary/MediaLibraryUtils.java

```diff
@@ -409,6 +408,9 @@ static void deleteAssets(Context context, String selection, String[] selectionAr
     } catch (SecurityException e) {
       promise.reject(ERROR_UNABLE_TO_SAVE_PERMISSION,
         "Could not delete asset: need WRITE_EXTERNAL_STORAGE permission.", e);
+    } catch (IllegalArgumentException e) {
+      e.printStackTrace();
+      promise.reject(ERROR_UNABLE_TO_DELETE, "Could not delete file.", e);
```

Comment:
We're trying to stick to `ERR` prefix for error names in the new code 😉 
```suggestion
      promise.reject(ERR_UNABLE_TO_DELETE, "Could not delete file.", e);
```

Reply:
I've just used the constant that already existed, but I can rename it for whole module 😉